utils.go

```diff
@@ -72,3 +72,9 @@ func logErrs(err []error) {
 		}
 	}
 }
+
+func debugLog(l string) {
```

Comment:
Ahah.  Well I'm not doing it  because the number of arguments and their type changes. It seems more scalable to interpolate the string first and just pass it as an argument.